crates/dslab-faas/src/cpu.rs

```diff
@@ -2,10 +2,10 @@
 /// We use something similar to CPUShares in cgroups, but instead of shares we operate with (shares/core_shares),
```

Comment:
```
/// CPU sharing models used in the simulator.
/// We use something similar to CPU shares in cgroups, but instead of shares we operate with (shares/core_shares),
/// i.e. if the container has 512 shares and each core amounts to 1024 shares, the share of the container equals 0.5.
/// If the container allows concurrent invocations, each invocation gets an equal part of the container share.
/// The exact CPU sharing model depends on the used CpuPolicy.
```

Reply:
Сделал